core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -21,7 +21,7 @@
  * The SQL and parameters parsed from an SQL statement.
  */
 public class ParsedSql {
-    private static final String POSITIONAL_PARAM = "?";
+    protected static final String POSITIONAL_PARAM = "?";
```

Comment:
This can be default/package access instead of protected

Reply:
Yes, it can be, but does not have to, because this variable is static. There is no value added in having this default-accessible.

Reply:
The added value is that package-protected is less visible than protected (which is accessible to package + subclasses).

Reply:
But there are no subclasses of `ParsedSql` due to a private nature of its only constructor. This class cannot be derived.

Reply:
Then why should be `protected` at all? :)

Reply:
For better readability.

Reply:
I don't agree... Please change it. Then we'll merge.

Reply:
Done. I'm making this one as resolved.